charts/seed-bootstrap/dashboards/seed-deployments-replicas.json

```
@@ -0,0 +1,432 @@
+        {
+          "exemplar": true,
+          "expr": "min(kube_deployment_status_replicas_updated{deployment=\"$deployment_name\"}) without (instance, pod)",
+          "hide": false,
+          "interval": "",
+          "intervalFactor": 2,
+          "legendFormat": "{{namespace}}: updated",
+          "refId": "D"
+        },
+        {
+          "exemplar": true,
+          "expr": "max(kube_deployment_spec_replicas{deployment=\"$deployment_name\"}) without (instance, pod)",
+          "hide": false,
+          "interval": "",
+          "intervalFactor": 2,
+          "legendFormat": "{{namespace}}: desired",
+          "refId": "E"
+        }
+      ],
+      "thresholds": [],
+      "timeRegions": [],
+      "title": "Replicas",
+      "tooltip": {
+        "shared": true,
+        "sort": 0,
+        "value_type": "individual"
+      },
+      "type": "graph",
+      "xaxis": {
+        "buckets": null,
+        "mode": "time",
+        "name": null,
+        "show": true,
+        "values": []
+      },
+      "yaxes": [
+        {
+          "format": "short",
+          "label": null,
+          "logBase": 1,
+          "max": null,
+          "min": null,
+          "show": true
+        },
+        {
+          "format": "short",
+          "label": null,
+          "logBase": 1,
+          "max": null,
+          "min": null,
+          "show": true
+        }
+      ],
+      "yaxis": {
+        "align": false,
+        "alignLevel": null
+      }
+    }
+  ],
+  "refresh": "",
+  "schemaVersion": 27,
+  "style": "dark",
+  "tags": [],
+  "templating": {
+    "list": [
+      {
+        "allValue": null,
+        "current": {
+          "selected": true,
+          "text": "auditlog-seed-controller-manager",
+          "value": "auditlog-seed-controller-manager"
+        },
+        "datasource": "seed-prometheus",
+        "definition": "label_values(kube_deployment_metadata_generation{namespace!=\"garden\", namespace!=\"kube-system\"}, deployment)",
+        "description": null,
+        "error": null,
+        "hide": 0,
+        "includeAll": false,
+        "label": "Deployment",
+        "multi": false,
+        "name": "deployment_name",
+        "options": [],
+        "query": {
+          "query": "label_values(kube_deployment_metadata_generation{namespace!=\"garden\", namespace!=\"kube-system\"}, deployment)",
+          "refId": "StandardVariableQuery"
+        },
+        "refresh": 2,
+        "regex": "",
+        "skipUrlSync": false,
+        "sort": 0,
+        "tagValuesQuery": "",
+        "tags": [],
+        "tagsQuery": "",
+        "type": "query",
+        "useTags": false
+      }
+    ]
+  },
+  "time": {
+    "from": "now-30m",
+    "to": "now"
+  },
+  "timepicker": {
+    "refresh_intervals": [
+      "10s",
+      "30s",
+      "1m",
+      "5m",
+      "15m",
+      "30m",
+      "1h",
+      "2h",
+      "1d"
+    ]
+  },
+  "timezone": "Seed Deployments Replicas",
+  "title": "",
```

This should probably be:
```suggestion
  "timezone": "",
  "title": "Seed Deployments Replicas",
```

---

Oh, my bad. 😅 
Thanks for pointing this.